packages/@sfcc-bff/basketapi/src/api/models/Basket.ts

```diff
@@ -44,7 +44,14 @@ class Basket {
                       price: product.price,
                       quantity: product.quantity,
                       itemId: product.itemId,
-                      image: product.image,
+                      imageURL: product.imageURL,
```

Comment:
I think we should be putting these values in the product API and use the Product model here.

Reply:
I agree with you. The `Basket.ts` needs to be cleaned up in entirety. Product details should stay in product module. That seems to be a bigger task.

Reply:
The problem is that the Basket.ts can not import Product.js model, the Image model is in the same productapi folder that's why SearchResultProduct.js can import Image model.

Reply:
then how are we able to use getPrices from productapi?

Reply:
After consulting with the team and the architect, the best approach is recommended to use the same data structure with SDK which has BasketT, ProductT and ProductItemT, but this requires the conversion from .js to .ts for productapi, also it might make sense to make the current Basket.ts class to be a function and move it to the basketresolver.ts. This should be done in a follow up story.